packages/core/js-client/src/connection/RelayConnection.ts

```diff
@@ -180,19 +180,27 @@ export class RelayConnection implements IConnection {
       );
     }
 
-    log.trace("sending particle...");
-
     // Reusing active connection here
     const stream = await this.lib2p2Peer.dialProtocol(
       this.relayAddress,
       PROTOCOL_NAME,
     );
 
-    log.trace("created stream with id ", stream.id);
+    log.trace(
+      "sending the particle %s to the created stream %s",
+      particle.id,
+      stream.id,
+    );
+
     const sink = stream.sink;
 
     await pipe([fromString(serializeToString(particle))], encode, sink);
-    log.trace("data written to sink");
+
+    log.trace(
+      "the particle %s have been written to the stream %s",
```

Comment:
```suggestion
      "particle %s sent to %s",
```
and relay address here, too

Reply:
and I think that's `debug`

Reply:
Yeap, `stream.id` seems redundant.
Not sure about relay address though, JS client always has a single relay address and it would spam the logs without particular benefit.

Reply:
imagine you have several js clients